File: uri/python/1008.py
employee_number = int(input())
worked_hours_in_a_month = int(input())
money_per_hour = float(input())

salary = worked_hours_in_a_month * money_per_hour

print('NUMBER =', employee_number)
print('SALARY = U$ {:.2f}'.format(salary))

File: uri/dart/1008.dart
import 'dart:io';

void main() {
  
  int employeeNumber = int.parse(stdin.readLineSync());
  int workedHours = int.parse(stdin.readLineSync());
  double moneyPerHour = double.parse(stdin.readLineSync());

  double salary = workedHours * moneyPerHour;

  print('NUMBER = $employeeNumber');
  print('SALARY = U\$ ${salary.toStringAsFixed(2)}');

}